packages/components/src/VirtualizedList/CellDatetime/CellDatetime.component.js

```diff
@@ -3,15 +3,36 @@ import React from 'react';
 import classnames from 'classnames';
 import { distanceInWordsToNow, format } from 'date-fns';
 import invariant from 'invariant';
+import enLocale from 'date-fns/locale/en';
+import frLocale from 'date-fns/locale/fr';
+import jaLocale from 'date-fns/locale/ja';
 
 import styles from './CellDatetime.scss';
+import { getCurrentLanguage } from '../../translate';
 
 const DATE_TIME_FORMAT = 'YYYY-MM-DD HH:mm:ss';
 
+export function getDateFnsLocale(language) {
+	switch (language) {
+		case 'en':
+			return enLocale;
+		case 'fr':
+			return frLocale;
+		case 'ja':
+			return jaLocale;
+		default:
+			return enLocale;
+	}
+}
+
 export function computeValue(cellData, columnData) {
 	try {
 		if (columnData.mode === 'ago') {
-			return distanceInWordsToNow(cellData, { addSuffix: 'ago' });
+			const currentLocale = getCurrentLanguage();
+			return distanceInWordsToNow(cellData, {
+				addSuffix: 'ago',
```

Comment:
I was wondering how `ago` was translated and noticed that it's supposed to be a boolean? https://date-fns.org/v1.27.0/docs/distanceInWordsToNow#arguments 
Can we please update that to `addSuffix: true`?